protostar/commands/test/test_runner.py

```diff
@@ -183,8 +183,8 @@ async def _invoke_test_case(
 
         if test_case.setup_fn_name:
             setup_case_result = await run_setup_case(test_case, state)
-            if isinstance(setup_case_result, FailedSetupCaseResult):
-                return setup_case_result.into_failed_test_case_result()
+            if isinstance(setup_case_result, BrokenSetupCaseResult):
+                return setup_case_result.into_broken_test_case_result()
```

Comment:
Off topic:
`into` is a code smell and can always be replaced with `from`. Change my mind.

@mkaput

Reply:
`from` should be preferred over `into` wherever possible, provided you know the target type

here we use `into` because we don't know the concrete type we convert into